### PR TITLE
Add announcement about the MapLibre User Group Japan

### PR DESCRIPTION
**Pull request by wipfli**

### CODE REVIEW

**Comment by wipfli** [comment]
Tweet would be:

> If you speak Japanese and you are interested in map rendering, have a look at the newly created MapLibre User Group Japan.

File: content/news/2023-01-23-maplibre-user-group-japan/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+<p>
+<img src="mlug-jp_logo.png" alt="Logo MapLibre User Group Japan" style="max-width:600px;width:100%">
+</p>
+
+Official Website: https://bit.ly/mug--jp
+
+Twitter Space: https://twitter.com/i/spaces/1OyKAVvlgvbGb
+
+Yasunori Kirimoto and Kanahiro Iguchi who are both [MapLibre Voting Members](https://github.com/maplibre/maplibre/blob/main/VOTING_MEMBERS.md) are organizing the User Group. The plan is to create online meetups and bring people together who are interested in MapLibre. They also have some nice MapLibre swag for you...
```
Comment (maxammann) [inline content/news/2023-01-23-maplibre-user-group-japan/index.md:18]:
```suggestion
Yasunori Kirimoto and Kanahiro Iguchi who are both [MapLibre Voting Members](https://github.com/maplibre/maplibre/blob/main/VOTING_MEMBERS.md) are organizing the User Group. The plan is to create online meetups and bring people together who are interested in MapLibre. They also have some nice MapLibre swag for you!
```

**Review by maxammann — APPROVED** [review]

**Review by lseelenbinder — APPROVED** [review]
And I like Max's change of `…` to `!`. 😄

**Review by HarelM — APPROVED** [review]

**Comment by wipfli** [comment]
https://twitter.com/maplibre/status/1617811187852533760

**Comment by wipfli** [comment]
https://www.linkedin.com/feed/update/urn:li:activity:7023578483003924480

**Pull request merged**